scripts/resolve-version-lib.js

```diff
@@ -68,6 +68,7 @@ function resolveVersion(rootdir) {
 
   return {
     version: currentVersion,
+    alphaVersion: require(versionFilePath).alphaVersion,
```

Comment:
It's maybe a bit nitpicky, but it feels weird to do the `require(versionFilePath)` twice. Odd spot of the code to prematurely optimize, but I think I'd prefer we only do the require once. We can store the value of the require above (~line 40), and then use it to grab the `currentVersion` and `alphaVersion`.

Reply:
Done 👍